<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayControl</class>
 <widget class="QWidget" name="PlayControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="9,1">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QLabel" name="label_play">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_open">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_close">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_continueMode">
        <property name="text">
         <string>连续采集</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_softTriggerMode">
        <property name="text">
         <string>软触发</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_grabbing">
        <property name="text">
         <string>采集</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_softTrigger">
        <property name="text">
         <string>软触发</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_saveBmp">
        <property name="text">
         <string>保存BMP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_saveJpg">
        <property name="text">
         <string>保存JPG</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
